### DISCUSSION TITLE
Check if any text entry is focused?

### DISCUSSION THREAD

**Discussion by lukors** [emilk/egui#1464] [Q&A] [answered]
I have hotkeys in my application. I want to turn those off when any text box is focused, so they don't activate when the user is writing.

How can I check if any text entry is focused in egui?

**Comment by emilk** [comment, marked as answer]
`ctx.memory().focus()` returns the widget that currently has keyboard focus, and thus `None` if no widget does.

However, it doesn't know if the widget is a text box or just a button you have tabbed to, or whatever.

**Reply by lukors** [reply]
Using that function did the trick for me, thanks! :)
